### PR TITLE
update file follow tuning docs to include event queue tuning

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: ingesters/file_follow.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The most common use case for File Follower is monitoring a directory containing
 
 Attention: On RHEL/CentOS, `/var/log` belongs to the "root" group, not "adm" as we assume. File Follower runs in the adm group by default, so if you want it to read `/var/log` you need to `chgrp -R adm /var/log` OR change the group in the systemd unit file.
 
+## Kernel parameter tuning
+
+On Linux, File Follower uses the inotify kernel subsystem. If you are seeing filesystem notification errors in File Follow, consider increasing the depth of the inotify event queue. You can do this by setting the depth with sysctl: `sysctl -w fs.inotify.max_queued_events=32768`.
```
Comment (traetox) [inline ingesters/file_follow.md:11]: can we call out the max watched files parameter too?


Just get it all down

**Review by traetox — CHANGES_REQUESTED** [review]
add in the max watched files parameter too

**Comment by djfritz** [comment]
@traetox PTAL

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**